packages/wdio-static-server-service/src/launcher.ts

```diff
@@ -44,7 +44,8 @@ export default class StaticServerLauncher {
         this.middleware.forEach(
             (ware: MiddleWareOption) => this.server.use(ware.mount, ware.middleware))
 
-        await promisify(this.server.listen.bind(this.server))(this.port)
+        const listenServer = this.server.listen.bind(this.server)
```

Comment:
is this implementation correct?

Reply:
Yes, but next line is missing await

Reply:
when using promisify, i am getting issue on next line.

![image](https://user-images.githubusercontent.com/45997477/95653694-21419980-0b18-11eb-8139-f104162167c0.png)

Reply:
Well, it should be used with promisify, investigate how to deal with that nicely. I believe we aren't the first ones trying that

Reply:
@mgrybyk adding the below code in launcher.ts file resolves the issue

declare module 'util' {
    function promisify<T1>(fn: (arg1: T1, callback: (err?: any) => void) => void): (arg1: T1) => Promise<void>;
}

The same has been suggested here :

https://github.com/microsoft/TypeScript/issues/26048#issuecomment-409973174

is this ok? or should i search for something else?

Reply:
can you set type for listen server variable instead of declaring module util?

See https://github.com/webdriverio/webdriverio/compare/ts-selenium-standlone-service-fix-types#diff-97c9d8dd32f5509001124f081429f24eR44

Reply:
Follow the above step, but getting the below error on runtime:

`packages/wdio-static-server-service/src/launcher.ts:46:15 - error TS2322: Type '(arg1: any, arg2: (() => void) | undefined) => Promise<any>' is not assignable to type '(port: number) => Promise<any>'.

    46         const listen: (port: number) => Promise<any> = promisify(this.server.listen)`